datahub-web-react/package.json

```diff
@@ -104,11 +102,7 @@
         "ec2-dev": "yarn run generate && CI=true;export CI;BROWSER=none craco start",
         "build": "yarn run generate && CI=false REACT_APP_MOCK=false craco build && rm -rf dist/ && cp -r build/yarn/ dist/ && rm -r build/yarn/",
         "test": "craco test",
-        "cy:run:ci": "cypress run",
         "pretest:e2e:ci": "yarn generate",
-        "test:e2e:ci": "start-server-and-test start:e2e 3010 cy:run:ci",
-        "cy:open": "cypress open",
-        "test:e2e": "start-server-and-test start:e2e 3010 cy:open",
```

Comment:
I think we can still keep the `test:e2e` command and call it out in the README but just remove the `cy:open` part.

So in this file: `"test:e2e": "start-server-and-test start:e2e 3010",`

then in the README:
```
In order to start a server and run frontend unit tests using react-testing-framework, run:

`yarn test :e2e`

There are also more automated tests using Cypress in the `smoke-test` folder of the repository root.
```